config/initializers/devise_security.rb

```diff
@@ -24,7 +24,6 @@
 
   # enable email validation for :secure_validatable. (true, false, validation_options)
   # dependency: see https://github.com/devise-security/devise-security/blob/master/README.md#e-mail-validation
-  config.email_validation = false
 
   # captcha integration for recover form
   # config.captcha_for_recover = true
```

Comment:
Can this file now be deleted following the updates that have been made?

Reply:
[29ec4b1](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1078/commits/29ec4b1812eb1250dd20d521f28ce96db1329b27)